eduVPN/ConnectionService.swift

```diff
@@ -261,40 +233,61 @@ class ConnectionService: NSObject {
             handler(.failure(Error.noHelperConnection))
             return
         }
-
+        
         let bundle = Bundle.init(for: ConnectionService.self)
         let openvpnURL = bundle.url(forResource: "openvpn", withExtension: nil, subdirectory: ConnectionService.openVPNSubdirectory)!
         let upScript = bundle.url(forResource: "client.up.eduvpn", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
         let downScript = bundle.url(forResource: "client.down.eduvpn", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
-        let leasewatchPlist = URL(fileURLWithPath: "/Library/Application Support/eduVPN/LeaseWatch.plist")
-        let leasewatchScript = bundle.url(forResource: "leasewatch", withExtension: "sh", subdirectory: ConnectionService.openVPNSubdirectory)!
         var scriptOptions = [
             "-6" /* ARG_ENABLE_IPV6_ON_TAP */,
             "-f" /* ARG_FLUSH_DNS_CACHE */,
             "-o" /* ARG_OVERRIDE_MANUAL_NETWORK_SETTINGS */,
             "-r" /* ARG_RESET_PRIMARY_INTERFACE_ON_DISCONNECT */,
-            "-w" /* ARG_RESTORE_ON_WINS_RESET */,
-            "-m" /* ARG_MONITOR_NETWORK_CONFIGURATION */,
-            "-t" /* ARG_TB_PATH */,
-            bundle.bundlePath
+            "-w" /* ARG_RESTORE_ON_WINS_RESET */
         ]
         let developerMode = preferencesService.developerMode
         if developerMode {
             scriptOptions.append("-l" /* ARG_EXTRA_LOGGING */)
         }
-       
+        
```

Comment:
There are many changes done in this file that shouldn't be done.

Reply:
Ok